src/harness/constraint_testcase.py

```diff
@@ -0,0 +1,125 @@
+#    Copyright 2016 SAS Project Authors. All Rights Reserved.
```

Comment:
There is no need to create a new type of testcase -- this can live in the MCP file directly.

Reply:
This   file is introduced for all MCP and related test cases, going forward we will   add all helper and utility methods in this file